lib/sync/remote-resin-io-device.coffee

```diff
@@ -142,13 +142,7 @@ exports.sync = ({ uuid, baseDir, destination, before, after, ignore, port = 22,
 
 		# Returns a promise that is resolved with the API-fetched device object or rejected on error or missing requirement
 		ensureDeviceRequirements = (device) ->
-			# Ensure user is the owner of the device. This is also checked on the backend side.
-			resin.auth.getUserId()
-			.then (userId) ->
-				if userId isnt device.user.__id
-					throw new Error('Resin sync is permitted to the device owner only. The device owner is the user who provisioned it.')
-			.then -> # Ensure minimum required HostOS version for resin sync
-				ensureHostOSCompatibility(device.os_version, MIN_HOSTOS_RSYNC)
```

Comment:
@pimterry yes, the user will get a generic 'device not found' error, because technically, the device will not be visible with his/her JWT token